resources/tentative/newssite/news-nuxt/app.vue

```diff
@@ -0,0 +1,39 @@
+<script setup>
+useHead({
+    title: "The Daily Broadcast",
+    htmlAttrs: {
+        lang: "en",
+    },
+    meta: [
+        { charset: "utf-8" },
+        { name: "viewport", content: "width=device-width, initial-scale=1" },
+        {
+            hid: "description",
+            name: "description",
+            content: "A news site developed with Nuxt.",
+        },
+    ],
+});
+</script>
+
+<script>
+  // "Fixes" an issue with calling history.replaceState too often in certain browsers during testing.
+  // Note: This should NOT be used in a production application.
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  history.replaceState = function(state) {
+    return null;
+  }
+
+  // This hack allows to capture the work normally happening in a rAF. We
+  // may be able to remove it if the runner improves.
+  window.requestAnimationFrame = (cb) => window.setTimeout(cb, 0);
+  window.cancelAnimationFrame = window.clearTimeout;
+</script>
+
+<template>
+  <NuxtLayout>
+    <NuxtPage />
+  </NuxtLayout>
```

Comment:
@danielroe - how strong do you feel about this?
The `default.vue` page is fairly large and I want to avoid an even larger `app.vue` file, unless there are some benefits associated with combining.

Reply:
It avoids the creation of an additional js chunk and dynamic import, which has a small overhead.